helpers/shortcode/dummy-image.js

```diff
@@ -0,0 +1,48 @@
+function template({
+  width,
+  height = width,
+  text = `${width}&#215;${height}`,
+  fontFamily = 'sans-serif',
+  fontWeight = 'bold',
+  fontSize = Math.floor(Math.min(width, height) * 0.2),
+  textAdjust = Math.floor(fontSize * 0.4),
+  bgColor = '#ddd',
+  textColor = 'rgba(0,0,0,0.5)'
+} = {}) {
+  return `<svg xmlns="http://www.w3.org/2000/svg" width="${width}" height="${height}" viewBox="0 0 ${width} ${height}">
+    <rect fill="${bgColor}" width="${width}" height="${height}"/>
+    <text fill="${textColor}" font-family="${fontFamily}" font-size="${fontSize}" dy="${textAdjust}" font-weight="${fontWeight}" x="50%" y="50%" text-anchor="middle">${text}</text>
+  </svg>`;
+}
+
+function encode(rendered) {
```

Comment:
Is this function just for en coding to base 64 images? If so it might be nice to have a slightly more verbose function name. e.g. `encodeToBase64`

Reply:
@Paul-Hebert It is not base 64 encoded, but I renamed the function so it was more clear what it was doing (I hope!)

Reply:
Ahh yeah that's a lot clearer. Thanks!